### ISSUE TITLE
Duplicate short option name in options

### ISSUE THREAD

**Issue by jkugler** [sensu-plugins/sensu-plugins-graphite#50]
In this file: https://github.com/sensu-plugins/sensu-plugins-graphite/blob/master/lib/sensu-plugins-graphite/graphite_proxy/options.rb

There are two config blocks that use the same short option: `-s`
```
         server: {
            description: 'Server host and port',
            short: '-s SERVER:PORT',
            long: '--server SERVER:PORT',
            required: true
```
and
```
          hostname_sub: {
            description: 'Character used to replace periods (.) in hostname (default: _)',
            short: '-s CHARACTER',
            long: '--host-sub CHARACTER'
          },
```
I would recommend keeping `-s` for server/port, and change the hostname sub short name.

**Comment by majormoses** [comment]
I think we should just remove the short version for that one, would you be willing to put together a pr for this. I am fine with this breaking change as it was really already broken. I will do a round of releases that way people can to it without the breaking change.

**Comment by majormoses** [comment]
closed via #53
